dynawo/sources/Modeler/DataInterface/IIDM/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ set(DATA_INTERFACE_IIDM_SOURCES
     DYNLccConverterInterfaceIIDM.cpp
     DYNHvdcLineInterfaceIIDM.cpp
     DYNServiceManagerInterfaceIIDM.cpp
+    ../DYNDataInterfaceIIDMCommon.cpp
```

Comment:
Please avoid relative paths in cmakes

Reply:
The issue is that the implem is valid for both IIDM library and then I try to make it common for both and not store it in the subfolders

Reply:
Then maybe it should be in the dynawo_DataInterface library?

Reply:
done